cmd/graph.go

```diff
@@ -94,6 +105,15 @@ func (gb *graphBuilder) cacheModule(m *Module) {
 	gb.modulesCache[moduleID(m.Version)] = m
 }
 
+func (gb *graphBuilder) getIndirectModuleFromMap(mod *module.Version) (*module.Version, bool) {
+	m, found := gb.indirectModuleMap[mod.Path]
+	return m, found
+}
+
+func (gb *graphBuilder) mapIndirectModule(m *module.Version) {
```

Comment:
QQ: why wrap this in a function?

Reply:
Probably because `getModuleFromCache` and `cacheModule` are defined the same way. Agreed it's not very useful - we can make this part of the next PR doing refactoring and adding unit tests..

Reply:
Yeah, I'd also add that it's not helping reading the code. We can address this later